gfx_2d_print.h

```diff
@@ -390,6 +390,30 @@ class Graphics2DPrint : public Graphics2D, public Print {
     }
     print(yourNumber);
   }
+  /**
+    @param str String parameter you nedd 'foo' -> E.g char foo[] = "index";
+    @param pos Nth count string
+    @param directionHead string cut head
+
+  */
+  void printSlice(char *str, int pos, bool directionHead = false) {
+    uint8_t len = strlen(str);
+
+    if (len > abs(pos)) {
+      if(directionHead){
+        if (pos > 0) 
+          str[pos] = 0;  // 3 : (Hell)o, World -> Hell
```

Comment:
@simonmicro 
How about this is code?

Reply:
* Maybe you should stay consistent inside the same function body - either using `0` XOR `'\0'`.
* You modify the input `str` variable. IIRC no other `print()` function needs to do that - how about you also try not to do that? This is especially relevant, as you can't use your `printSlice()` function for any `const char*` calls (e.g. from program-memory).